worlds/civ_6/Civ6Client.py

```diff
@@ -0,0 +1,338 @@
+import asyncio
+import logging
+import os
+import traceback
+from typing import Dict, List
+import zipfile
+
+from CommonClient import ClientCommandProcessor, CommonContext, get_base_parser, logger, server_loop, gui_enabled
+from .Data import get_progressive_districts_data
+from .DeathLink import handle_check_deathlink
+from NetUtils import ClientStatus
+import Utils
+from .CivVIInterface import CivVIInterface, ConnectionState
+from .Enum import CivVICheckType
+from .Items import CivVIItemData, generate_item_table, get_item_by_civ_name
+from .Locations import generate_era_location_table
+from .TunerClient import TunerErrorException, TunerTimeoutException
+
+
+class CivVICommandProcessor(ClientCommandProcessor):
+    def __init__(self, ctx: CommonContext):
+        super().__init__(ctx)
+
+    def _cmd_deathlink(self):
+        """Toggle deathlink from client. Overrides default setting."""
+        if isinstance(self.ctx, CivVIContext):
+            self.ctx.death_link_enabled = not self.ctx.death_link_enabled
+            self.ctx.death_link_just_changed = True
+            Utils.async_start(self.ctx.update_death_link(
+                self.ctx.death_link_enabled), name="Update Deathlink")
+            self.ctx.logger.info(
+                f"Deathlink is now {'enabled' if self.ctx.death_link_enabled else 'disabled'}")

```

Comment:
I don't think there's something in here for it, so you should make it update the client's tags when you enable/disable death link.
https://github.com/ArchipelagoMW/Archipelago/blob/main/docs/network%20protocol.md#connectupdate

Reply:
Ah let me know if I'm looking at the wrong thing but it looks like the `update_death_link` call actually takes care of that:
https://github.com/hesto2/Archipelago/blob/civ6-1.0/CommonClient.py#L628
![image](https://github.com/user-attachments/assets/6e9111fd-9469-46a0-8383-0f2eff277b00)

Reply:
Ah I'll have to look where that's getting set 👀

Reply:
Looks like that is set by default for the common client, although I think qwint has a PR up to make it so that is no longer set by default